modules/ccs-gcp-customer-requirements.adoc

```diff
@@ -11,7 +11,7 @@
 [id="ccs-gcp-requirements-account_{context}"]
 == Account
 
-* The customer ensures that link:https://cloud.google.com/storage/quotas[Google Cloud limits] are sufficient to support {product-title} provisioned within the customer-provided GCP account.
+* The customer ensures that link:https://cloud.google.com/storage/quotas[Google Cloud limits] and link:https://cloud.google.com/compute/resource-usage[allocation quotas that apply to Compute Engine] are sufficient to support {product-title} provisioned within the customer-provided GCP account.
```

Comment:
There are some Red Hats that could do with a {nbsp} on lines 22 and 34 of this doc if you're so inclined. :)